index.js

```diff
@@ -2017,6 +2017,24 @@
     return Foldable.methods.reduce(foldable)(f, x);
   }
 
+  //# foldMap :: (Monoid m, Foldable f) => (TypeRep m, a -> m, f a) -> m
+  //.
+  //. Deconstructs a foldable by mapping every element to a monoid and
+  //. concatenating the results.
+  //.
+  //. This function is derived from [`reduce`](#reduce), [`concat`](#concat)
+  //. and [`empty`](#empty).
```

Comment:
Our convention is to list functions in the order in which they appear in the file: `concat`, `empty`, `reduce`, in this case. Please also add a [serial comma][1] when rearranging these items.


[1]: https://en.wikipedia.org/wiki/Serial_comma